cypresse2e/cypress/elements/pages/aboutUs.js

```diff
@@ -0,0 +1,28 @@
+class AboutUs {
+  linkedInButton = "#AU_LN";
+  facebookButton = "#AU_FB";
+  instagramButton = "#AU_IG";
+  twitterButton = "#AU_TW";
+  getInvolvedCloseButton = ".ant-modal-close";
+  getInvolvedButton = "#AU_GI";
+  joinAsVolunteerButton = "#AU_GI_JV";
+  joinAsVolunteerLink = "https://apply.workable.com/fightpandemics";
+  joinAsAmbassadorButton = "#AU_GI_JA";
+  joinAsAmbassadorLink = "https://apply.workable.com/fightpandemics/j/46D6EF3B44/";
+  joinStudentProgramButton = "#AU_GI_JSP";
+  joinStudentProgramLink = "https://apply.workable.com/fightpandemics/j/58B157AAB2/";
+  getToHelpBoardButton = "#AU_VHB";
+  goToHelpBoardLink = "/feed";
+  lifetimeSupportersContainerXpath = "//*[@id=\"root\"]/div/div/div[3]/main/div/div[8]/div[1]";
+  currentSupportersContainerXpath = "//*[@id=\"root\"]/div/div/div[3]/main/div/div[8]/div[2]";
+  pastSupportersContainerXpath = "//*[@id=\"root\"]/div/div/div[3]/main/div/div[8]/div[3]";
+  lifetimeNumberOfMembers = 11;
+  currentNumberOfMembers = 35;
+  pastNumberOfMembers = 2;
```

Comment:
@TimeaKovacs86 I know @galeza had mentioned earlier that we can also check the members but the number of supporters will likely keep increasing overtime and also move between the lifetime/current/past sections. I think just checking that the supporter containers exist should be enough so that we don't have to keep updating the number in when new ones are added.

Reply:
@jessica-lau @TimeaKovacs86 
ok, agree